ssp

```diff
@@ -8353,6 +8355,55 @@ CPU: $cpu w/ $cores core(s)
 END_OF_TEMPLATE
 }
 
+sub print_doc_bug_report {
+    my $version = $CPANEL_VERSION ? $CPANEL_VERSION : 'Unknown';
+    my $ticket = "";
+    if ( defined $ENV{HISTFILE} and $ENV{HISTFILE} =~ /ticket.(\d+)$/ ) {
+        $ticket = $1;
+    }
+    print <<END_OF_TEMPLATE
+==== BEGIN COPY BELOW ====
+{panel:title=(i) Document and ticket URLS}
+   
+* Link to the document that requires modification.
+* Link to the support ticket that contains the relevant information.
+   
+{panel}
+   
+----
+   
+{panel:title=(+) Details}
+   
+Provide relevant information for any documentation changes.
+If this is a request for new documentation, provide an outline with the technical details required to complete the document.
+  
+   
```

Comment:
Closing panel tag is missing here.  There are also spaces on empty lines in the template.  The Wiki page containing the template has been updated to address this, please also update this pull request with the changes.